app-5-rdf-serializer/src/main/java/org/geovistory/toolbox/streams/rdf/processors/project/ProjectEntityRdfsLabel.java

```diff
@@ -0,0 +1,137 @@
+                    var newRdfValueOp = Utils.booleanIsEqualTrue(newValue.getDeleted$1()) ? Operation.delete : Operation.insert;
+                    var newRdfValue = ProjectRdfValue.newBuilder().setOperation(newRdfValueOp).build();
+                    var newRdfKey = ProjectRdfKey.newBuilder().setProjectId(aggKey.getProjectId())
+                            .setTurtle("<" + GEOVISTORY_RESOURCE.getUrl() + aggKey.getEntityId() + "> <" + RDFS.getUrl() + "label> \"" + newValue.getLabel() + "\"@^^<" + XSD.getUrl() + "string> .")
+                            .build();
+                    var newRdfRecord = ProjectRdfRecord.newBuilder().setKey(newRdfKey).setValue(newRdfValue).build();
+
+
+                    // if: size 2 -> we remove the first item from list, as it is not relevant anymore.
+                    if (aggValue.getList().size() == 2) {
+                        aggValue.getList().remove(0);
+                    }
+
+                    // case: size 1
+                    if (aggValue.getList().size() == 1) {
+                        var oldVal = aggValue.getList().get(0).getValue();
+                        // if operation was delete, remove it
+                        if (oldVal.getOperation() == Operation.delete) {
+                            aggValue.getList().remove(0);
+                        }
+                        // else we modify the operation from 'insert' to 'delete'
+                        else {
+                            oldVal.setOperation(Operation.delete);
+                        }
+                    }
+                    // case: size 0 -> do nothing
+
+                    // add the new rdf record to the agg value
+                    aggValue.getList().add(newRdfRecord);
+
+                    return aggValue;
+                },
+                Materialized.<ProjectEntityKey, ProjectRdfList, KeyValueStore<Bytes, byte[]>>as("aggregated-project-entity-rdfs-label-store") /* state store name */
+                        .withKeySerde(
+                                avroSerdes.ProjectEntityKey()
+                        )
+                        .withValueSerde(
+                                avroSerdes.ProjectRdfList()
+                        )
+        );
+
+        var projectRdfStream = aggregatedStream.toStream().flatMap(
+                (key, value) -> {
+                    List<KeyValue<ProjectRdfKey, ProjectRdfValue>> result = new LinkedList<>();
+                    value.getList().forEach(projectRdfRecord -> {
+                        result.add(KeyValue.pair(projectRdfRecord.getKey(), projectRdfRecord.getValue()));
+                    });
+                    return result;
+                }
+        );
+
+
+        /* SINK PROCESSORS */
+
+        projectRdfStream.to(outputTopicNames.projectRdf(),
+                Produced.with(avroSerdes.ProjectRdfKey(), avroSerdes.ProjectRdfValue())
+                        .withName(outputTopicNames.projectRdf() + "-class-label-producer")
```

Comment:
"-class-label-producer" should be "-project-entity-label-producer" (we can't have two producers with the same name)

Reply:
done